.changeset/nine-ducks-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+"@itwin/presentation-components": minor
+---
+
+Replaced `react-select` with `iTwinUI` comboboxes, limited visible options to 100, and added a non-pickable option prompting users to filter beyond the limit.
```

Comment:
I think a change of this size requires a more detailed description. E.g. something like this:

```markdown
Replaced `react-select` with [iTwinUI's ComboBox](https://itwinui.bentley.com/docs/combobox).

This affects [list the public components that this affects] components.

Changes:

- The number of select options is limited to 100. When more items exist, a non-selectable option is displayed at the bottom of the list, prompting users to provide an items filter. Previously, additional pages of select options was loaded when user scrolled to the bottom of the list.

- Did the style change? A before/after screenshot would be great.

- Maybe something else?
```

Reply:
Rewritten.